.gitignore

```diff
@@ -60,6 +61,7 @@ keys/
 # easy sharing, see:
 # <https://code.visualstudio.com/docs/editor/extension-marketplace#_workspace-recommended-extensions>
 !.vscode/extensions.json
+.vscode
```

Comment:
We can add developer-setup-specific patterns that are unrelated to the repository to e.g. the global .gitignore. This is a general and official recommendation, see <https://git-scm.com/docs/gitignore#_description>.

Reply:
Thanks - DONE. here is the supporting article https://blog.martinhujer.cz/dont-put-idea-vscode-directories-to-projects-gitignore/